docs/authentication/using-middleware.mdx

```diff
@@ -11,38 +11,40 @@ Because Payload uses your existing Express server, you are free to add whatever
 This approach has a ton of benefits - it's great for isolation of concerns and limiting scope, but it also means that your additional routes won't have access to Payload's user authentication.
 
 <Banner type="success">
-  You can make full use of Payload's built-in authentication within your own custom Express endpoints by adding Payload's authentication middleware.
+  You can make full use of Payload's built-in authentication within your own
+  custom Express endpoints by adding Payload's authentication middleware.
 </Banner>
 
 Example in `server.js`:
+
 ```ts
-import express from 'express';
-import payload from 'payload';
+import express from "express";
+import payload from "payload";
 
 const app = express();
 
 payload.init({
-  secret: 'PAYLOAD_SECRET_KEY',
-  mongoURL: 'mongodb://localhost/payload',
+  secret: "PAYLOAD_SECRET_KEY",
+  mongoURL: "mongodb://localhost/payload",
   express: app,
 });
 
 const router = express.Router();
 
+// Note: Payload must be initialized before the `payload.authenticate` middleware can be used
```

Comment:
The comment is definitely useful, but I think the issue has happened enough to merit a notice on the middleware page here: https://payloadcms.com/docs/authentication/using-middleware 

(Unless this already adds a visible note to that page) @jacobsfletch